### PR TITLE
Make EFT Path in settings clearer

### PR DESCRIPTION
**Pull request by artehe**
Should hopefully make the EFT path setting much clearer, and also harder to select the BSG EFT install by mistake now.

I think this is the nicest way of displaying it and doing it but am open to any suggestions.

image

### CODE REVIEW

File: SIT.Manager/ViewModels/Settings/EftViewModel.cs [hunk 1]
```diff
@@ -1,27 +1,46 @@
...
         _barNotificationService = barNotificationService;
+        _installerService = installerService;
         _localizationService = localizationService;
         _versionService = versionService;
 
+        BsgEftInstallPath = Path.GetDirectoryName(_installerService.GetEFTInstallPath()) ?? "Unable to detect BSG EFT install...";
+        SitEftInstallPath = _configsService.Config.SitEftInstallPath;
+        if (string.IsNullOrEmpty(SitEftInstallPath))
+        {
+            SitEftInstallPath = "SIT install not found. Please point to existing install here, or install by clicking the install button on the left";
```
Comment (Plootie) [inline SIT.Manager/ViewModels/Settings/EftViewModel.cs:41]: Should this be localized?

**Review by Plootie — COMMENTED** [review]
Just confirmation on my comment, otherwise looks good

File: SIT.Manager/ViewModels/Settings/EftViewModel.cs (hunk 1 above)
Comment (artehe) [reply SIT.Manager/ViewModels/Settings/EftViewModel.cs:41]: oh ye, there's probably a couple other places in this I might have missed 😓

Comment (artehe) [reply SIT.Manager/ViewModels/Settings/EftViewModel.cs]: Should have got all of them now.

**Review by Plootie — APPROVED** [review]

**Pull request merged**